internal/coreinternal/processor/filterlog/filterlog.go

```diff
@@ -92,6 +103,9 @@ func (mp *propertiesMatcher) MatchLogRecord(lr pdata.LogRecord, resource pdata.R
 	if lr.Body().Type() == pdata.ValueTypeString && mp.bodyFilters != nil && mp.bodyFilters.Matches(lr.Body().StringVal()) {
 		return true
 	}
+	if mp.severityFilters != nil && mp.severityFilters.Matches(lr.SeverityText()) {
```

Comment:
Log "severity" is technically a number, whereas "severity text" is a string. 

I think we can anticipate that filtering will be necessary on numeric severity as well. Would that fit into this configuration option or be separate. If separate, should this one be called something else to avoid the future naming collision?

Reply:
OK addressed.